core/src/main/scala/cromwell/util/DatabaseUtil.scala

```diff
@@ -0,0 +1,22 @@
+package cromwell.util
+
+import java.sql.SQLTransientException
+
+import akka.actor.ActorSystem
+import cromwell.core.retry.{Retry, SimpleExponentialBackoff}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+object DatabaseUtil {
+  private def isTransient(throwable: Throwable): Boolean = throwable match {
```

Comment:
In this case I don't know if it would add a lot of value though because this method's only purpose is to be passed to the withRetry